packages/codegen/src/__tests__/codegen.uint.spec.ts

```diff
@@ -95,7 +94,7 @@ describe('code generation', () => {
           }, {});
       };
 
-      const base = '/api/v1/projects/{project}';
```

Comment:
the generated URLs from OpenAPI has changed. Assuming this is expected, have adjusted the base URLs of codegen to account for this